config/locales/views/moderations/en.yml

```diff
@@ -53,6 +53,14 @@ en:
             Expert: Expert
             Mid-level: Mid-level
             Novice: Novice
+        admin:
+          heading: Admin actions
+          heading2: Higher permission post options
+          subtitle: Higher permission post options
+          aria_label: Open admin actions
+          icon: Crown
+          desc_html: Who <em>might</em> find this post most valuable, based on overall experience level?
+          from: "%{lvl} - "
```

Comment:
I might be just missing something, but it doesn't look to me like these keys are being used?

Reply:
Yeah, these ones are left over from a copy/paste job. Removed.